std/math.d

```diff
@@ -5495,22 +5493,6 @@ nothrow @nogc:
                                  | inexactException,
         }
     }
-    else version(ARM)
```

Comment:
These two branches are not equal. The first branch is missing the `subnormalException` definition.

Reply:
Someone doesn't read commit messages.

Reply:
Nope :)